src/components/pages/JudgeTable/JudgeTable.js

```diff
@@ -148,14 +152,12 @@ export default function JudgeTable(props) {
           postJudge(bookmarks[i]);
         }
       }
+    } else {
+      alert('Please selected judge(s) to be saved');
     }
     alert('Judges Successfully Saved');
```

Comment:
To keep it consistent, maybe change to "Judge(s) Successfully Saved"

Reply:
Good catch!! I went ahead and made those changes. Thanks for your suggestions!